frontend/Controls/CopyButton.qml

```diff
@@ -0,0 +1,57 @@
+import QtQuick 2.7
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.3
+
+Label {
+    id: copyPasteButton
```

Comment:
This seems to be a copy of the "copyPasteButton" in `frontend/XCITE/ReceiveCoins/Form.qml` near line 30.  Can you change `frontend/XCITE/ReceiveCoins/Form.qml` to use the refactored button control as well and make sure it tests OK?

Also, the naming of this control isn't consistent with the other buttons - it should probably be changed to `ButtonCopy.qml`.